JuliaInterface/src/gap_macros.c

```diff
@@ -23,6 +24,9 @@ Obj gap_julia(jl_value_t * julia_obj)
     if (IsGapObj(julia_obj)) {
         return (Obj)(julia_obj);
     }
+    if (jl_typeis(julia_obj,JULIA_GAPFFE_type)){
```

Comment:
This is still not clang-formatted. I now made PR #110 to help us automatically track these things, *if* we can agree that's a good thing. If you rather not be forced to clang-format your commits, I can also shut up about it.

Reply:
Sorry, I just sometimes forget. I think the formatting is a really good thing and should be enforced!